src/test/Makefile

```diff
@@ -667,7 +670,11 @@ $(OBJECT_DIR)/serial_msp_unittest : \
 	$(OBJECT_DIR)/serial_msp_unittest.o \
 	$(OBJECT_DIR)/gtest_main.a
 
+ifeq ($(UNAME), Darwin)
+	$(CXX) $(CXX_FLAGS) -Wl,$(TEST_DIR)/parameter_group.ld $^ -o $(OBJECT_DIR)/$@
```

Comment:
what does the -no_pie flag do and why doesn't this one need it when the others do?